backstop/tests/menu.html

```diff
@@ -703,298 +717,340 @@ <h3>Select</h3>
-                class="iui-icon"
-                aria-hidden="true"
-              ></svg-placeholder>
-              <span class="iui-content">
-                <div class="iui-menu-label">Beta</div>
-                <div class="iui-menu-description">Lorem ipsum</div>
-              </span>
-              <svg-caret-right-small
-                class="iui-icon"
-                aria-hidden="true"
-              ></svg-caret-right-small>
-            </li>
-            <li
-              class="iui-menu-item iui-large"
-              tabindex="0"
-            >
-              <svg-placeholder
-                class="iui-icon"
-                aria-hidden="true"
-              ></svg-placeholder>
-              <span class="iui-content">
-                <div class="iui-menu-label">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-                  tempor
-                  incididunt ut labore et
-                  dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-                  aliquip ex
-                  ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
-                  fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
-                  deserunt
-                  mollit anim id est laborum.</div>
-                <div class="iui-menu-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                  eiusmod
-                  tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
-                  ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
-                  voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
-                  proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
-              </span>
-              <svg-caret-right-small
-                class="iui-icon"
-                aria-hidden="true"
-              ></svg-caret-right-small>
-            </li>
-            <li
-              class="iui-menu-divider"
-              role="separator"
-            ></li>
-            <li
-              class="iui-menu-item"
-              tabindex="0"
+            <svg
+              viewBox="0 0 16 16"
+              xmlns="http://www.w3.org/2000/svg"
+              aria-hidden="true"
             >
-              <span class="iui-content">
-                <div class="iui-menu-label">My projects</div>
-              </span>
-            </li>
-          </ul>
+              <path
+                d="m4.8067 6h6.3953a.27961.27961 0 0 1 .24043.44321l-3.1736 3.45707a.33969.33969 0 0 1 -.48085 0l-3.2217-3.45707a.26909.26909 0 0 1 .24042-.44321z"
+              ></path>
+            </svg>
+          </span>
         </div>
+        <ul
```

Comment:
I think this menu needs to be outside the `label` to fix the responsive grid issue. However, I think that's just because we are not using a popover here in the html demo. In React, it should work fine.

Reply:
Fixed